web/beacon-app/src/features/topics/utils.tsx

```diff
@@ -0,0 +1,30 @@
+import { t } from '@lingui/macro';
+
+export const getDefaultTopicStats = () => {
+  return [
+    {
+      name: t`Online Publishers`,
+      value: 0,
+      units: '',
+    },
+    {
+      name: t`Online Subscribers`,
+      value: 0,
+      units: '',
+    },
+    {
+      name: t`Avg Events/ Second`,
```

Comment:
During office hours, it was stated that this will be the total number of events and not events per second.